packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -1283,403 +1257,96 @@ describe("toMapTree", () => {
-						schemaValidationPolicy,
-					);
-				});
-
-				it("Failure", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode(true);
-					assert.throws(
-						() =>
-							mapTreeFromNodeData(
-								content,
-								[myMapSchema, schemaFactory.string],
-								new MockNodeIdentifierManager(),
-								schemaValidationPolicy,
-							),
-						outOfSchemaExpectedError,
-					);
-				});
-			});
-
-			describe("Array node", () => {
-				const content = ["foo"];
-				const fieldSchema = getFieldSchema(FieldKinds.required, [
-					brand(schemaFactory.string.identifier),
-				]);
-				const myArrayNodeSchema = schemaFactory.array("myArrayNode", [schemaFactory.string]);
-
-				function createSchemaAndPolicyForMapNode(invalid: boolean = false) {
-					return createSchemaAndPolicy(
-						new Map<TreeNodeSchemaIdentifier, TreeNodeStoredSchema>([
-							[
-								// An invalid stored schema will associate the string identifier to a number schema
-								brand(schemaFactory.string.identifier),
-								invalid
-									? new LeafNodeStoredSchema(ValueSchema.Number)
-									: new LeafNodeStoredSchema(ValueSchema.String),
-							],
-							[brand(myArrayNodeSchema.identifier), new MapNodeStoredSchema(fieldSchema)],
-						]),
-						new Map([[fieldSchema.kind, FieldKinds.required]]),
-					);
-				}
-				it("Success", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode();
-					mapTreeFromNodeData(
-						content,
-						[myArrayNodeSchema, schemaFactory.string],
-						new MockNodeIdentifierManager(),
-						schemaValidationPolicy,
-					);
-				});
-
-				it("Failure", () => {
-					const schemaValidationPolicy = createSchemaAndPolicyForMapNode(true);
-					assert.throws(
-						() =>
-							mapTreeFromNodeData(
-								content,
-								[myArrayNodeSchema, schemaFactory.string],
-								new MockNodeIdentifierManager(),
-								schemaValidationPolicy,
-							),
-						outOfSchemaExpectedError,
-					);
-				});
-			});
-		});
-
-		const schemaValidationPolicyForSuccess = createSchemaAndPolicy(
-			new Map([
-				[brand(schemaFactory.string.identifier), new LeafNodeStoredSchema(ValueSchema.String)],
-			]),
-			new Map(),
-		);
-		const schemaValidationPolicyForFailure = createSchemaAndPolicy(
-			new Map([
-				[
-					// Fake a stored schema that associates the string identifier to a number schema
-					brand(schemaFactory.string.identifier),
-					new LeafNodeStoredSchema(ValueSchema.Number),
-				],
-			]),
-			new Map(),
-		);
-
-		describe("cursorFromInsertable", () => {
-			it("Success", () => {
-				cursorFromInsertable(schemaFactory.string, "Hello world", nodeKeyManager);
-			});
-
-			it("Failure", () => {
-				assert.throws(
-					() =>
-						// @ts-expect-error invalid data for schema
-						cursorFromInsertable(schemaFactory.number, "Hello world", nodeKeyManager),
-					validateUsageError(/incompatible/),
-				);
-			});
+	describe("getPossibleTypes", () => {
```

Comment:
This suite is actually unchanged other than indent: it was incorrectly under "Stored schema validation" as has been un-nested